### PR TITLE
chore(actions): move to actions from travis

### PR DESCRIPTION
**Pull request by nellyk**
Travis build are failing when trying to release -switching this main and github actions to try and resolve this

### CODE REVIEW

File: .github/workflows/health-check.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [ '10.x', '12.x', '14.x' ]
+    name: Node ${{ matrix.node }}
+    steps:
+    - uses: actions/checkout@v2
+    - run: |
+        git remote set-branches --add origin main alpha
```
Comment (mtomcal) [inline .github/workflows/health-check.yml:18]: Do we need `alpha` for this library?

**Review by mtomcal — CHANGES_REQUESTED** [review]

File: .github/workflows/health-check.yml (hunk 1 above)
Comment (nellyk) [reply .github/workflows/health-check.yml:18]: oops copy pasta

Comment (nellyk) [reply .github/workflows/health-check.yml]: Thanks for highlighting this

**Review by JamesSingleton — APPROVED** [review]

**Review by mtomcal — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -94,7 +91,20 @@
       "@semantic-release/git",
       "@semantic-release/github"
     ],
-    "branch": "master"
+    "branches": [
+      "+([0-9])?(.{+([0-9]),x}).x",
```
Comment (JAdshead) [inline package.json:95]: what branches should this be running on ?

File: package.json [hunk 3]
```diff
@@ -94,7 +91,20 @@
...
+      "main",
+      "next",
```
Comment (JAdshead) [inline package.json:97]: Not sure we want to trigger a release on next or next-major

**Review by JAdshead — CHANGES_REQUESTED** [review]

**Review by JAdshead — APPROVED** [review]

File: .github/workflows/tests.yml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        node-version: ${{ matrix.node }}
+    - name: Install Dependencies
+      run: npm ci
+      env:
+        NODE_ENV: development
+    - name: Unit Tests
+      run: npm run test:unit
+      env:
+        NODE_ENV: production
+        GITHUB_TOKEN: ${{ secrets.PA_TOKEN }}
```
Comment (Francois-Esquire) [inline .github/workflows/tests.yml:32]:
```suggestion
```

File: .github/workflows/tests.yml [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    - name: Git History Test
+      run: npm run test:git-history
+      env:
+        NODE_ENV: production
+        GITHUB_TOKEN: ${{ secrets.PA_TOKEN }}
```
Comment (Francois-Esquire) [inline .github/workflows/tests.yml:37]:
```suggestion
```

File: .github/workflows/tests.yml [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+    - name: Lockfile Lint Test
+      run: npm run test:lockfile
+      env:
+        NODE_ENV: production
+        GITHUB_TOKEN: ${{ secrets.PA_TOKEN }}
```
Comment (Francois-Esquire) [inline .github/workflows/tests.yml:42]:
```suggestion
```

File: .github/workflows/tests.yml [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+    - name: Lint
+      run: npm run test:lint
+      env:
+        NODE_ENV: production
+        GITHUB_TOKEN: ${{ secrets.PA_TOKEN }}
```
Comment (Francois-Esquire) [inline .github/workflows/tests.yml:47]:
```suggestion
```

**Review by Francois-Esquire — CHANGES_REQUESTED** [review]
Small security nitpick... we probably shouldn't be exposing the PA_TOKEN if we don't need to

**Review by infoxicator — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (nellyk) [reply package.json:97]: Hopefully this was answered by @JamesSingleton  we are overriding semantic release method to ensure that we can use main branch. This are the inbuilt functions

**Review by JamesSingleton — APPROVED** [review]

**Pull request merged**